src/lib/docker/dockerService.ts

```diff
@@ -347,6 +349,24 @@ class DockerService implements DockerLibrary {
       }
     }
   }
+
+  /**
+   * Renames the docker volume directory on disk when a node is renamed
+   * @param network the network containing the node
+   * @param node the node that's to be renamed
+   * @param newName the new name for the node and directory
+   */
+  async updateDirs(network: Network, node: CommonNode, newName: string) {
+    const volumeDir = (node as AnyNode).implementation
+      .toLocaleLowerCase()
+      .replace('-', '');
+    const oldPath = join(network.path, 'volumes', volumeDir, node.name);
+    const newPath = join(network.path, 'volumes', volumeDir, newName);
+
+    fs.rename(oldPath, newPath)
+      .then(() => console.log(`Renamed directory from ${oldPath} to ${newPath}`))
```

Comment:
You should use `info()` or `debug()` so the log message is saved in the log file.